### ISSUE TITLE
XMRig for esp32 

### ISSUE THREAD

**Issue by cinwie** [xmrig/xmrig#3549]
Is it possible to run XMRig on esp32 ? If anyone can help please let me know.

**Comment by geekwilliams** [comment]
I don't believe so.  The ESP is a microcontroller.  Xmrig runs on top of a full OS like Linux or Windows.

**Comment by TekuSP** [comment]
Absolutely no. But I am sure there are some miners for ESP32, just not XMRig
